README.md

```diff
@@ -6,6 +6,71 @@
 
 ### Welcome to the .NET **nanoFramework** Hardware.Esp32.Rmt Library repository
 
+RMT (Remote Control) is an ESP32 module driver that is, originally, intended to be used with infrared remote control signals. However, the module and APIs are generic enough that they can used to send/receive other types of signals.
+
+## Getting Started
+
+Our samples repository contains commented code showcasing how to use the RMT module in ESP32 MCUs to control various types of devices using nanoFramework. The RMT samples can be found [here](https://docs.espressif.com/projects/esp-idf/en/v4.4.3/esp32/api-reference/peripherals/rmt.html).
```

Comment:
```suggestion
Our samples repository contains commented code showcasing how to use the RMT module in ESP32 MCUs to control various types of devices using nanoFramework. A detailed explanation about the RMT module can be found [here](https://docs.espressif.com/projects/esp-idf/en/v4.4.3/esp32/api-reference/peripherals/rmt.html).
```

Reply:
Apologies, I pasted the wrong link here. I actually wanted to include both.